mozilla_vpn/dashboards/vpn_saasboard_retention.dashboard.lookml

```diff
@@ -1264,143 +1493,6 @@
-    point_style: none
-    show_value_labels: false
-    label_density: 25
-    x_axis_scale: auto
-    y_axis_combined: true
-    ordering: none
-    show_null_labels: false
-    show_totals_labels: false
-    show_silhouette: false
-    totals_color: "#808080"
-    y_axes: [{label: '', orientation: left, series: [{axisId: retention_rate, id: 2019-10
-              - retention_rate, name: 2019-10}, {axisId: retention_rate, id: 2019-11
-              - retention_rate, name: 2019-11}, {axisId: retention_rate, id: 2019-12
-              - retention_rate, name: 2019-12}, {axisId: retention_rate, id: 2020-01
-              - retention_rate, name: 2020-01}, {axisId: retention_rate, id: 2020-02
-              - retention_rate, name: 2020-02}, {axisId: retention_rate, id: 2020-03
-              - retention_rate, name: 2020-03}, {axisId: retention_rate, id: 2020-04
-              - retention_rate, name: 2020-04}, {axisId: retention_rate, id: 2020-05
-              - retention_rate, name: 2020-05}, {axisId: retention_rate, id: 2020-06
-              - retention_rate, name: 2020-06}, {axisId: retention_rate, id: 2020-07
-              - retention_rate, name: 2020-07}, {axisId: retention_rate, id: 2020-08
-              - retention_rate, name: 2020-08}, {axisId: retention_rate, id: 2020-09
-              - retention_rate, name: 2020-09}, {axisId: retention_rate, id: 2020-10
-              - retention_rate, name: 2020-10}, {axisId: retention_rate, id: 2020-11
-              - retention_rate, name: 2020-11}, {axisId: retention_rate, id: 2020-12
-              - retention_rate, name: 2020-12}, {axisId: retention_rate, id: 2021-01
-              - retention_rate, name: 2021-01}, {axisId: retention_rate, id: 2021-02
-              - retention_rate, name: 2021-02}, {axisId: retention_rate, id: 2021-03
-              - retention_rate, name: 2021-03}, {axisId: retention_rate, id: 2021-04
-              - retention_rate, name: 2021-04}, {axisId: retention_rate, id: 2021-05
-              - retention_rate, name: 2021-05}, {axisId: retention_rate, id: 2021-06
-              - retention_rate, name: 2021-06}, {axisId: retention_rate, id: 2021-07
-              - retention_rate, name: 2021-07}, {axisId: retention_rate, id: 2021-08
-              - retention_rate, name: 2021-08}], showLabels: true, showValues: true,
-        unpinAxis: false, tickDensity: default, tickDensityCustom: 5, type: linear}]
-    defaults_version: 1
-    hidden_fields: [original_subscriptions__retention.subscription_count, total_retention_rate]
-    series_types: {}
-    hidden_pivots: {}
-    listen:
-      Country: subscriptions.country_name
-      Pricing Plan: subscriptions.pricing_plan
-      Provider: subscriptions.provider
-      Plan Interval Type: subscriptions.plan_interval_type
-      Product Name: subscriptions.product_name
-      Original Subscription Start Date: subscriptions.original_subscription_start_date
-    row: 58
-    col: 0
-    width: 24
-    height: 7
   filters:
```

Comment:
It doesn't look like any filters for plan ID or fraudulent charges were added to this retention dashboard.  Is that intentional?

Reply:
No it is not.. Let me check if they got lost during my battle with git.

Reply:
The default values for the filters start from line#1627